packages/shared_preferences/lib/shared_preferences.dart

```diff
@@ -84,10 +84,16 @@ class SharedPreferences {
   void _setValue(String valueType, String key, Object value) {
```

Comment:
Would probably be cool to document that `null` removes a key in the doc comments of the methods.

Reply:
Good idea